diff --git a/CMake/HPHPFindLibs.cmake b/CMake/HPHPFindLibs.cmake
index 52f0b7a..9127139 100644
--- a/CMake/HPHPFindLibs.cmake
+++ b/CMake/HPHPFindLibs.cmake
@@ -203,4 +203,8 @@ macro(hphp_link target)
 	target_link_libraries(${target} ${ONIG_LIB})
 	target_link_libraries(${target} ${Mcrypt_LIB})
 	target_link_libraries(${target} ${GD_LIBRARY})
+
+	target_link_libraries(${target} crypt)
+	target_link_libraries(${target} rt)
+	target_link_libraries(${target} resolv)
 endmacro()
