--- hiphop-php-HPHP-2.1.0/hphp/runtime/vm/jit/phys-reg.h~	2013-07-12 21:00:55.000000000 +0300
+++ hiphop-php-HPHP-2.1.0/hphp/runtime/vm/jit/phys-reg.h	2013-10-06 02:59:16.294268856 +0300
@@ -250,8 +250,13 @@
   uint64_t m_bits;
 };
 
+#if 1
+static_assert(std::is_trivially_destructible<RegSet>::value,
+              "RegSet must have a trivial destructor");
+#else
 static_assert(std::has_trivial_destructor<RegSet>::value,
               "RegSet must have a trivial destructor");
+#endif
 
 //////////////////////////////////////////////////////////////////////
 
