--- hhvm-HHVM-3.3.0/CMake/HPHPIZEFunctions.cmake~	2014-09-19 19:10:19.000000000 +0300
+++ hhvm-HHVM-3.3.0/CMake/HPHPIZEFunctions.cmake	2014-09-21 23:28:42.031159399 +0300
@@ -18,6 +18,12 @@
   set_target_properties(${EXTNAME} PROPERTIES PREFIX "")
   set_target_properties(${EXTNAME} PROPERTIES SUFFIX ".so")
   install(TARGETS ${EXTNAME} DESTINATION "${CMAKE_INSTALL_LIBDIR}/hhvm/extensions/${HHVM_API_VERSION}")
+  install(TARGETS ${EXTNAME} DESTINATION "lib/hphp/extensions/${HHVM_API_VERSION}")
+  if(HHVM_EXTENSION_DIR)
+    install(TARGETS ${EXTNAME} DESTINATION ${HHVM_EXTENSION_DIR})
+  else()
+    install(TARGETS ${EXTNAME} DESTINATION "${CMAKE_INSTALL_LIBDIR}/hhvm/extensions/${HHVM_API_VERSION}")
+  endif()
 endfunction()
 
 # Add an extension that uses the Zend compatibility layer.
